examples/react-native/legacyDataMigration/MigrationChecker.ts

```diff
@@ -0,0 +1,133 @@
+import {Event, UserSession} from '@amplitude/analytics-types';
+
+type LegacyEvent = {
+  event_id: number;
+  event_type: string;
+  time: number;
+  insert_id: string;
+};
+
+const legacyDeviceId = '22833898-c487-4536-b213-40f207abdce0R';
```

Comment:
Can you please check sessionId too

Reply:
I didn't check it because sessionId/lastEventTime are hard-coded in database content to some date in May 23 and therefore new session is started in tests.
Now fixed - I changed hard-coded sessionId/lastEventTime to future date (2088 year), new session is not started now.